src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -84,21 +84,24 @@ export class SchoolSelectField extends React.Component {
 
   handleOptionClick = ({ address1, address2, address3, city, facilityCode, name, state, zip, country }) => {
     this.props.selectInstitution({ address1, address2, address3, city, facilityCode, name, state });
-    const backendFriendlyCountries = {
-      USA: 'United States'
+    const isDomesticAddress = country === 'USA';
+    const address = {
```

Comment:
Maybe put this transformation into a helper.

Reply:
Nice. Will do.